packages/editor-ui/src/utils/objectUtils.ts

```diff
@@ -0,0 +1,14 @@
+export function isObjectOrArray(maybeObject: unknown): maybeObject is { [key: string]: string } {
+	return typeof maybeObject === 'object' && maybeObject !== null;
+}
+
+export function isObject(maybeObject: unknown): maybeObject is { [key: string]: string } {
```

Comment:
Can we make it a generic to avoid locking into `string` type?

```ts
export function isObject<K = string, V = string>(maybeObject: unknown): maybeObject is Record<K, V> {
```